db/seeds.rb

```diff
@@ -8,7 +8,8 @@
   {name: 'Canon'},
   {name: 'Sony'},
   {name: 'Gitzo'},
-  {name: 'Lee'}
+  {name: 'Lee'},
+  {name: 'Zeiss'}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.